keps/2724-topology-aware-scheduling/README.md

```diff
@@ -1048,17 +1048,17 @@ Consequently, it selects the nodes with 1, 2, 3, and 3 available pods, reserving
 
 Selection of the algorithm depends on TAS profiles expressed by feature gates, and PodSet's annotation:
 
-| featuregate/annotation                   | preferred         | required          | unconstrained     |
+| feature gate / annotation                | preferred         | required          | unconstrained     |
 | ---------------------------------------- | ----------------- | ----------------- | ----------------- |
-| None                                     | BestFit           | BestFit           | BestFit           |
-| TASProfileMixed (deprecated)             | BestFit           | BestFit           | LeastFreeCapacity |
+| TASProfileMixed (default)                | BestFit           | BestFit           | LeastFreeCapacity |
+| TASProfileBestFit (deprecated)           | BestFit           | BestFit           | BestFit           |
 | TASProfileLeastFreeCapacity (deprecated) | LeastFreeCapacity | LeastFreeCapacity | LeastFreeCapacity |
 
-Feature gates: `TASProfileMixed` and `TASProfileLeastFreeCapacity` are mutually exclusive.
+These 3 feature gates are mutually exclusive. Enabling neither of them is equivalent to enabling TASProfileMixed.
 
-We recommend the BestFit algorithm for most of use cases, however we give more flexibility to users with those experimental feature gates.
+We recommend the default profile for most of use cases, however we give more flexibility to users with those experimental feature gates.
 Based on the collected feedback we will introduce TAS configuration that would allow user to select the desired algorithm.
-Eventually we'll remove the feature as they will be no longer need when we implement API for TAS configuration.
+Eventually we'll remove the feature gates as they will be no longer need when we implement API for TAS configuration.
```

Comment:
Please add a note clearly indicating there is a change of the default in 0.15. We need to make sure it is communicated to the users.

Maybe "Until 0.15 the default mode in was `TASProfileBestFit` without any feature gate. However, based on the user feedback we decided to change to default to `TASProfileMixed` which only differs in case of the "unconstained" topology request. However, in this case Kueue should prioritize minimizing the fragmentation which is provided by the `LeastFreeCapacity` algorithm.
In order to introduce a bailout option to some users we introduce the TASProfileBestFit as deprecated. We will remove it in 0.17 if no report indicating a need for that configuration."